internal/pkg/service/buffer/store/model/column/column_test.go

```diff
@@ -110,3 +110,23 @@ func TestColumn_Header(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, string(headerMarshalled), val)
 }
+
+func TestColumn_Template(t *testing.T) {
+	t.Parallel()
+
+	// c := column.Template{Language: column.TemplateLanguageJsonnet, Content: `headers['Content-Type']`}
+	c := column.Template{Language: column.TemplateLanguageJsonnet, Content: "body.key1"}
+
+	body := orderedmap.New()
+	body.Set("key1", "val1")
+	body.Set("key2", "val2")
+
+	header := http.Header{}
+	header.Set("Content-Type", "application/json")
+	header.Set("Content-Encoding", "gzip")
+
+	val, err := c.CsvValue(column.ImportCtx{Body: body, Header: header})
+	assert.NoError(t, err)
```

Comment:
@michaljurecko Tomu asi nerozumím. Jak `not implemented`? 😅
![image](https://user-images.githubusercontent.com/215660/204340337-d5d35808-d257-46d7-bcb1-43673bdd5ff2.png)

Jde to tady z toho switche: https://github.com/google/go-jsonnet/blob/0504998e1e0c17217c222e8f541694c74bdf3c86/interpreter.go#L299 a fakt tam `ast.Object` není. 😕

Reply:
Ale je tam nějakej `*ast.DesugaredObject`. To by mohlo být ono, možná jsem použil jen špatný typ.

Reply:
Jj ono to robí nejakú pred-kompilaciu.

Reply:
(ak by to nešlo napis)